packages/components/date-picker/src/date-picker-com/panel-date-pick.vue

```diff
@@ -513,6 +513,7 @@ export default defineComponent({
     }
 
     const isValidValue = (date_) => {
```

Comment:
We should add `unknown` type, and check if `date_`(I suggest it should be renamed to `date`) is `dayjs`
```ts
date instanceof dayjs
```

Reply:
Maybe we should add `Dayjs` type ?